app/views/public/event_detail.py

```diff
@@ -84,7 +84,9 @@ def display_event_detail_home(identifier):
             sponsors[int(sponsor.level)] = [sponsor]
 
     fees = DataGetter.get_fee_settings_by_currency(event.payment_currency)
-    code = request.args.get("code")
+    acode = request.args.get("acode")
```

Comment:
`Allow the combination of access + discount code links (manually for now) e.g. /e/47582707/?acode=VIP&dcode=888` 

it was just an example.